PJ05/src/Request/CreateConversationResponse.java

```diff
@@ -0,0 +1,17 @@
+package Request;
+
+import Exceptions.RequestFailedException;
+
+import java.util.UUID;
+
+public class CreateConversationResponse extends Response {
+    UUID conversation_uuid;
+
+    public CreateConversationResponse(boolean state, String msg, UUID request_uuid) {
+        super(state, msg, request_uuid);
+    }
+
+    public CreateConversationResponse(boolean state, String msg, UUID request_uuid, RequestFailedException exception) {
+        super(state, msg, request_uuid, exception);
+    }
```

Comment:
You need to include `conversation_uuid` in the constructor. Also, you should not include `exception` in the constructor because it's needed only when a request fails.

Reply:
Also, all fields must be `public final` so the Client can read them.